Resources/Prototypes/Entities/Objects/Weapons/Guns/Battery/battery_guns.yml

```diff
@@ -107,6 +107,33 @@
     proto: RedLaser
     fireCost: 62.5
 
+- type: entity
+  name: syndicate laser gun
+  parent: BaseWeaponBattery
+  id: WeaponSyndieLaserCarbine
+  description: A laser gun that's equipped with some expensive modifications that allows it to recharge slowly. 
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Guns/Battery/moddedlasergun.rsi
+    layers:
+    - state: base
+      map: ["enum.GunVisualLayers.Base"]
+    - state: mag-unshaded-4
+      map: ["enum.GunVisualLayers.MagUnshaded"]
+      shader: unshaded
+  - type: Item
+    sprite: Objects/Weapons/Guns/Battery/moddedlasergun.rsi
+  - type: Gun
+    selectedMode: SemiAuto
+    availableModes:
+    - SemiAuto
+  - type: HitscanBatteryAmmoProvider
+    proto: RedLaser
+    fireCost: 71.4285714286 
```

Comment:
Is there a reason this value is calculated to the 10-billionth place?

Reply:
> Is there a reason this value is calculated to the 10-billionth place?

blame math, honestly, 14 doesn't fit into 1000 too nicely